packages/inter-protocol/src/proposals/startPSM.js

```diff
@@ -138,14 +138,11 @@ export const startPSM = async (
     }),
   );
 
-  const governedInstance = await E(governorFacets.creatorFacet).getInstance();
-  const creatorFacet = E(governorFacets.creatorFacet).getCreatorFacet();
-
-  psmInstanceR.resolve(governedInstance);
+  psmInstanceR.resolve(await E(governorFacets.creatorFacet).getInstance());
```

Comment:
with all these facets, consider having a single entity in the space that is the result of `zoe.startInstance`.

Reply:
yeah; we tried that... ran into problems around promises or something... the answer isn't obvious, so we agreed to defer.